express.js

```diff
@@ -2,16 +2,16 @@
 // customising the .env file in your project's root folder.
 require('dotenv').load();
 
-var express = require('express');
+let express = require('express');
 
-var middleware = require('./routes/middleware');
+let middleware = require('./routes/middleware');
 
-var expressValidator = require('express-validator');
-var bodyParser = require('body-parser');
-var passport = require('passport'),
-	OAuthStrategy = require('passport-oauth').OAuth2Strategy;
+let expressValidator = require('express-validator');
```

Comment:
Since we're targeting node 9+, it's safe to use `const` -- best do so for `require`s.